frontend/appflowy_flutter/packages/appflowy_ui/lib/src/component/menu/menu.dart

```diff
@@ -11,7 +11,9 @@ class AFMenu extends StatelessWidget {
     super.key,
     required this.children,
     this.width,
+    this.builder,
```

Comment:
It would be better to create a new menu widget to support `ListView.builder` instead of modifying the existing one.